documentation/source/simulator_support.rst

```diff
@@ -89,8 +89,7 @@ Verilator
     Therefore, Verilator support in cocotb is currently experimental.
     Some features of cocotb may not work correctly or at all.
 
-    **Currently cocotb only supports Verilator 4.106 (no earlier or later version).**
-    See also the corresponding cocotb issue :issue:`2300` and `upstream issue <https://github.com/verilator/verilator/issues/2778>`_.
+    ** cocotb only supports Verilator 5.006 and later.**
```

Comment:
```suggestion
    **cocotb only supports Verilator 5.006 and later.**
```
The space breaks bold formatting:
https://cocotb--3231.org.readthedocs.build/en/3231/simulator_support.html#verilator